libs/klv/src/main/java/org/codice/ddf/libs/klv/data/text/KlvEncodingDetectedString.java

```diff
@@ -47,9 +47,12 @@ protected void decodeValue(Klv klv) {
     CharsetDetector charsetDetector = new CharsetDetector();
     charsetDetector.setText(bytes);
     CharsetMatch charsetMatch = charsetDetector.detect();
+
     try {
-      value = new String(bytes, charsetMatch.getName());
-      return;
+      if (charsetMatch != null) {
```

Comment:
✏️  Not a fan of this if within the try which makes the try completely useless if charsetMatch is null. How about a check to see if it is null before the try and returning `new String(bytes, Charset.defaultCharset())`?